### ISSUE TITLE
help

### ISSUE THREAD

**Issue by JCURB** [theabbie/Youtube-Uploader-Bot#11]
what can i use as the link? or where can i get a video?

**Comment by theabbie** [comment]
@JCURB you need to use download link of video, which directly links to the .mp4 file, and the video will be automatically locked by YouTube on upload because app isn't verified, you can see the video in youtube studio.

**Comment by JCURB** [comment]
how do i get the download link can i use mediafire?

**Comment by theabbie** [comment]
@JCURB if opening the link downloads the video, you can use it, mediafire might work if it directly opens the raw video.

**Comment by JCURB** [comment]
how can i make a link like that

**Comment by theabbie** [comment]
@JCURB If you have a link like this https://www.w3schools.com/html/mov_bbb.mp4 which directly opens the video or downloads it, you can use it, you can get the download link from mediafire.

**Comment by JCURB** [comment]
so media fire works?

**Comment by JCURB** [comment]
it still doesnt work

**Comment by JCURB** [comment]
![Uploading 2D3D5A9E-C317-465D-958A-BC85B9F32E0F.jpeg…]()

**Comment by JCURB** [comment]
also i can i have #s in the title?

**Comment by JCURB** [comment]
https://www.mediafire.com/file/vrscpdjxcr2mx3w/IMG_0221.MP4/file

**Comment by JCURB** [comment]
that works?

**Comment by theabbie** [comment]
@JCURB you can have #s in title, mediafire should work if you have download URL, try some other video link, the one I gave above for testing, and check YouTube studio to see if upload was successful.

**Comment by theabbie** [comment]
@JCURB that won't work, this will work, https://download2176.mediafire.com/o78x6wlz3ghg/vrscpdjxcr2mx3w/IMG_0221.MP4

**Comment by JCURB** [comment]
https://download2176.mediafire.com/ci2xllhq49dg/vrscpdjxcr2mx3w/IMG_0221.MP4

**Comment by JCURB** [comment]
thats the link

**Comment by JCURB** [comment]
yet it still wont upload

**Comment by theabbie** [comment]
@JCURB the one I posted should work, try it

**Comment by JCURB** [comment]
ive tried doesnt upload

**Comment by theabbie** [comment]
@JCURB what's the error?

**Comment by JCURB** [comment]
it doesnt show an error on the google code thing

**Comment by theabbie** [comment]
@JCURB if all steps have been followed, it should work, you were prompted to login with your Google account right?

**Comment by JCURB** [comment]
yes

**Comment by JCURB** [comment]
![Uploading E1B06A5F-FA29-4BAE-BDDC-B417A53A7FA8.jpeg…]()

**Comment by theabbie** [comment]
@JCURB the images you posted aren't visible, anyways, the video will be locked anyways, so no point getting troubled.

**Comment by JCURB** [comment]
what do i do?

**Comment by JCURB** [comment]
how can i unlock it

**Comment by theabbie** [comment]
@JCURB there is no good way to upload public videos to YouTube, regarding this error, I don't know where you're stuck, try this link https://www.w3schools.com/html/mov_bbb.mp4 if it doesn't upload, maybe this method is broken.

**Comment by JCURB** [comment]
that works

**Comment by theabbie** [comment]
@JCURB great, so mediafire is the issue, try some other source

**Comment by JCURB** [comment]
what other source

**Comment by JCURB** [comment]
[#](https://www.dropbox.com/s/28f9on01gnqofx1/Video%2026-02-2022%2C%2008%2017%2017.mp4?dl=0)

**Comment by theabbie** [comment]
@JCURB anywhere where you get download link, maybe Google drive can also work.

**Comment by theabbie** [comment]
@JCURB Dropbox won't work because in opening the link you don't get video but a webpage with download link, you need just the raw video.

**Comment by JCURB** [comment]
tried that didnt work

**Comment by JCURB** [comment]
hello?

**Comment by JCURB** [comment]
is it possible if i send you the vid and you can get some code together and make it upload that

**Comment by theabbie** [comment]
@JCURB not possible using code, you need correct link of video.

**Comment by JCURB** [comment]
is there a way you can make me the link

**Comment by theabbie** [comment]
@JCURB no, maybe just get a proper download link, also YouTube will lock video so point trying this.

**Comment by JCURB** [comment]
so its not working?

**Comment by theabbie** [comment]
@JCURB it is working, your links are the issue.
